### PR TITLE
Joomla! 3.9.26

### PR DESCRIPTION
**Pull request by zero-24**

### CODE REVIEW

File: 2021/26xxx/CVE-2021-26030.json [hunk 1]
```diff
@@ -1,17 +1,63 @@
 {
```
Comment (cve-team) [inline 2021/26xxx/CVE-2021-26030.json:1]: File name CVE ID and JSON CVE ID do not match.

File: 2021/26xxx/CVE-2021-26031.json [hunk 2]
```diff
@@ -1,17 +1,63 @@
 {
```
Comment (cve-team) [inline 2021/26xxx/CVE-2021-26031.json:1]: File name CVE ID and JSON CVE ID do not match.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by zero-24** [comment]
Just updated the IDs.

**Pull request merged**